packages/ui/src/components/widget/index.tsx

```diff
@@ -480,15 +487,7 @@ const SwapWidget: FC<SwapWidgetProps> = ({
                   direction: 'input',
                   token_symbol: token.symbol
                 })
-                if (
```

Comment:
Same about conditionally removing it because we probably want to keep the logic of switching, unless the address is different.